tst/testall.g

```diff
@@ -0,0 +1,6 @@
+LoadPackage("digraphs");
+if DigraphsTestInstall() and DigraphsTestAll() and DigraphsTestExtreme() then
+  FORCE_QUIT_GAP(0);
+fi;
+
+FORCE_QUIT_GAP(1);
```

Comment:
Same comment as above?

Reply:
Yes. Although if this line has been reached, then something went wrong, and that something could have gone badly wrong, and so I think it makes sense to be as drastic as possible when quitting GAP.

Reply:
As mentioned above, `QuitGap` is a new function in the GAP `master` branch, not available in any released versions, so we can't use it until Digraphs requires the upcoming GAP 4.12 or later.